### DISCUSSION TITLE
Infinite loop on Media getUrl

### DISCUSSION THREAD

**Discussion by labomatik** [spatie/laravel-medialibrary#3095] [Q&A]
Hello,

I'm experiencing strange behavior since an update on the aws/aws-sdk-php package.
Here is the stack trace:

Xdebug has detected a possible infinite loop, and aborted your script with a stack depth of '256' frames

![image](https://user-images.githubusercontent.com/179987/202136980-4760511b-cb87-4785-a2b0-a5a458b77993.png)

All files are stored on a S3 like (space @ DigitalOcean)

I've narrowed down the issue to a particular release of aws/aws-sdk-php
"aws/aws-sdk-php": "3.242.1", -> OK
"aws/aws-sdk-php": "3.243.0", -> NOT OK

Do you have any idea what could be wrong?

**Comment by parallels999** [comment]
>Do you have any idea what could be wrong?

No idea
Try to rollback aws/aws-sdk-php package to the working version, and try to find what did change there
